devel/parallel-hashmap/Portfile

```diff
@@ -23,6 +23,7 @@ patchfiles-append       0001-Warn-on-4-byte-char-not-err-out.patch \
                         0002-Unbreak-build-on-Apple-with-GCC-do-not-use-clangism.patch
 
 compiler.cxx_standard   2011
+compiler.thread_local_storage yes
```

Comment:
This looks enabled for all platforms. If changing compiler options, this means that the port contents would be different, and whenever this is the case, that usually necessitates a `revision` bump.

Reply:
> This looks enabled for all platforms. If changing compiler options, this means that the port contents would be different, and whenever this is the case, that usually necessitates a `revision` bump.

I think it is not needed, since on later systems compilers support TLS, so nothing will change for them. Same applies to PowerPC (which uses new GCC).